tests/test_add_area.py

```diff
@@ -48,5 +45,17 @@ def _add_area():
     return area_mbid
 
 
+def test_add_area(mb_client, browser, reset_db):
+    try:
+        area_mbid = _add_area(mb_client, browser)
```

Comment:
Would probably make sense to also test the area has all the bits you wanted to add? :) Might be enough to just query the area page and check the page displays the data (that's how we usually test these things in MB)

Reply:
Oops. Looks like I forgot to include all the test 🤦 
On it :)

Reply:
As far as I can tell, this is still not testing that the values got added properly, just that the area itself did get added :)

Reply:
Whooops. Just defeated the whole purpose of a test lmao. Fixing it ASAP

Reply:
Fixed (finally) :D